packages/ui-components/src/Common/Switch/index.module.css

```
@@ -0,0 +1,59 @@
+@reference "../../styles/index.css";
+
+.switch {
+  @apply inline-flex
+    justify-end
+    gap-3;
+
+  .label {
+    @apply cursor-pointer
+      select-none
+      text-sm
+      font-medium
+      text-neutral-800
+      dark:text-neutral-200;
+  }
+
+  .root {
+    @apply w-10.5
+      relative
+      inline-flex
+      h-6
+      cursor-pointer
+      items-center
+      rounded-full
+      bg-black
+      transition-colors
+      duration-100
+      ease-out
```

```suggestion
      motion-safe:transition-colors
      motion-safe:duration-100
      motion-safe:ease-out
```

As a best practice for accessibility, it would be beneficial to apply animations using `motion-safe` https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html